.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
-vendor
\ No newline at end of file
+vendor
+composer.lock
```

Comment:
composer.lock should still be versioned, could you revert this ?

Reply:
composer.lock is useless for library, so I removed it. Version information should be specified in composer.json.

Reply:
Actually, it could be useful to test against fixed versions of the dependencies, see https://getcomposer.org/doc/02-libraries.md#lock-file

Reply:
`However, this lock file will not have any effect on other projects that depend on it. It only has an effect on the main project.` If some projects depend on `phpokeapi`, the dependence can be not the same one for library test. So it can be not worked for expect.

Reply:
You got a point, let's keep the composer.lock in the .gitignore file, since it hasn't any effect

Reply:
ok